digiwf-engine/digiwf-engine-service/src/main/java/de/muenchen/oss/digiwf/legacy/mailing/process/SendMailTemplateDelegate.java

```diff
@@ -65,6 +65,7 @@ public void execute(final DelegateExecution delegateExecution) throws Exception
                 .receivers(receivers)
                 .subject(subject)
                 .body(emailBody)
+                .htmlBody(true)
```

Comment:
Wenn ich mir den alten Code so ansehe, dann wurde dort beim Text auch auf dem `MimeMessageHelper` auch kein `HTML=true` gesetzt.

Wie hat das vorher funktioniert mit HTML?

Will nur sichergehen, dass wir hier nicht noch etwas kaputt machen und verstehen, wie es zu dem Fehler gekommen ist, um es zukünftig zu verhindern.

Reply:
Das war mein Fehler beim refactoring / email lib erstellen. Ich habe das überall angepasst nur nicht in den 2 DelegateTasks.

Ganz ursprünglich wurden alle Mails in der Engine als HTML geschickt (das war hardcoded). Die Email Integration schickt aber Text Emails. Nachdem ich die Lib aber in beidem nehmen wollte habe ich das konfigurierbar gemacht und eben an den 2 Stellen vergessen zu konfigurieren 😅

Reply:
```java
final MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, true, "UTF-8");
helper.setText(text, true);
```

Davor wurden standardmäßig immer html Mails rausgeschickt. Ich habe das nur an den 2 Stellen in den Delegates vergessen zu konfigurieren.
Die Email Integration schickt aber nur Text Mails, weswegen das jetzt konfigurierbar ist.